acceptance/tests/api-gateway/api_gateway_v2_test.go

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package apigateway
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+
+	"github.com/hashicorp/consul-k8s/acceptance/framework/consul"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/helpers"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/k8s"
+	"github.com/hashicorp/consul-k8s/acceptance/framework/logger"
+	meshv2beta1 "github.com/hashicorp/consul-k8s/control-plane/api/mesh/v2beta1"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// Test that api gateway basic functionality works in a default installation and a secure installation for V2
+func TestAPIGateway_V2_Basic(t *testing.T) {
```

Comment:
do we need this in both the api-gateway and api-gateway-v2 directories?

Reply:
Nope. Missed deleting that, good catch